### PR TITLE
Fix Cassiopeia mobile breakpoint too small

### PR DESCRIPTION
**Pull request by RickR2H**
Pull Request for Issue # .

### Summary of Changes
The Cassiopeia template used mobile breakpoint that are to small. With this PR the breakpoint to switch to mobile is shifted from 768px to 992px. My initial PR #34687 to make it breakpoints fully configurable got rejected and I hope this PR is a solution to fix the majority of mobile issues.

breakpoint

The fact that the mobile breakpoint is to small is probably introduced to the switch from Bootstrap 4 to Bootstrap 5 which introduces a sifting in breakpoints.

### Testing Instructions
Install the PR and rebuild the CSS. Best to use the demo data to check all other elements too. See if the template behaves like expected.

### Actual result BEFORE applying this Pull Request
Mobile breakpoint to small

### Expected result AFTER applying this Pull Request
Mobile breakpoint a bit bigger

### Documentation Changes Required
No

### CODE REVIEW

**Comment by barsdeveloper** [comment]
There was no shifting in breakpoints. They just added a new breakpoint, the other ones behave exactly as they did.
https://getbootstrap.com/docs/5.0/migration/#grid-updates

On my device xs is vertical mobile, sm is horizontal mobile. md is never hit.

**Comment by RickR2H** [comment]
> There was no shifting in breakpoints. They just added a new breakpoint, the other ones behave exactly as they did.
> https://getbootstrap.com/docs/5.0/migration/#grid-updates
> 
> On my device xs is vertical mobile, sm is horizontal mobile. md is never hit.

Never mind the breakpoints comment.
On mobile the breakpoints will never be met because the screen is always to small 😉

**Comment by RickR2H** [comment]
@barsdeveloper to clarify a bit more:

mobile-view

**Comment by RickR2H** [comment]
@bembelimen is this PR a better solution?

**Comment by bembelimen** [comment]
Yes, something like that should be the way forward, thx.

**Comment by chmst** [comment]
Looks good for me. Tesing in firefox,win 10.  
Would be nice to get tests from different browsers and screen widths.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34780">issues.joomla.org/tracker/joomla-cms/34780</a>.</sub>

**Comment by brianteeman** [comment]
@RickR2H can you fix the conflicts please

**Comment by RickR2H** [comment]
@brianteeman @chmst Conflicts are fixed

File: administrator/templates/atum/scss/blocks/_login.scss [hunk 1]
```diff
@@ -219,8 +219,8 @@
       }
     }
 
-    @include media-breakpoint-down(md) {
-      background: var(--template-bg-dark-70);
+    @include media-breakpoint-down(lg) {
+      background: var(--atum-bg-dark-70);
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_login.scss]:
```suggestion
      background: var(--template-bg-dark-70);
```

Comment (RickR2H) [reply administrator/templates/atum/scss/blocks/_login.scss]: @Quy Thanks for the title change 😉 Regarding your proposed change. The CSS custom property is indeed missing. Nice catch. Will add the change to the PR.

**Comment by MSnoeren** [comment]
I have tested this item :white_check_mark: successfully on c3201d8f7468337ba1e1fc681bd4473c203be940<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34780">issues.joomla.org/tracker/joomla-cms/34780</a>.</sub>

**Comment by ricardo1709** [comment]
I have tested this item :white_check_mark: successfully on c3201d8f7468337ba1e1fc681bd4473c203be940<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34780">issues.joomla.org/tracker/joomla-cms/34780</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34780">issues.joomla.org/tracker/joomla-cms/34780</a>.</sub>

**Comment by drmenzelit** [comment]
@RickR2H why are you changing Atum file too?

**Comment by RickR2H** [comment]
@drmenzelit That wasn't my intention. Reverted to SCSS of the Atum template.

**Comment by bembelimen** [comment]
Thx

**Comment by testantinos** [comment]
Is there a way to make the hamburger menu to appear from 992px using css override on user. css? I manage to do so only by changing original template css from @media 768 to @ media 992? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34780">issues.joomla.org/tracker/joomla-cms/34780</a>.</sub>

**Comment by drmenzelit** [comment]
> Is there a way to make the hamburger menu to appear from 992px using css override on user. css? I manage to do so only by changing original template css from @media 768 to @ media 992?
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/34780](https://issues.joomla.org/tracker/joomla-cms/34780).

You can create an override of the menu and change the class controlling the breakpoint. Have a look at Bootstrap.

**Comment by testantinos** [comment]
Thanks for your help. So this way  I can create the code to include in my user.css or the procedure is deferent? I am new with bootstrap...<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34780">issues.joomla.org/tracker/joomla-cms/34780</a>.</sub>

**Comment by RickR2H** [comment]
Please use this stream only if you have comment regarding the pull request!
As addition to the previous comment, search ik Google for "How to create a Joomla override" This will het jou started. If you really need help, then it's better to ask for help via the Joomla forums.

**Pull request merged**